### ISSUE TITLE
Allow filling the edges of a selection

### ISSUE THREAD

**Issue by haykam821** [EngineHub/WorldEdit#2279]
### The Problem

Currently, the `//faces` command exists to fill the six faces of a cuboid region. However, there is no way to specify to only fill the outer edges of this cuboid region.

### A Solution

A complement command could be introduced with the syntax `//edges <pattern>`. For a cuboid region, this would fill its twelve edges. The behavior for other regions is more difficult to define, but the main purpose of this command would be to use it with cuboid regions.

### Alternatives

The closest solution I know if is using `//faces` then manually changing the difference between faces and edges back to air.

**Comment by TomyLobo** [comment]
How about `//g magenta_concrete (abs(x)==1)+(abs(y)==1)+(abs(z)==1)>=2`?
![image](https://user-images.githubusercontent.com/963797/224513193-fb4e28ea-957d-4c0e-9be5-747da2bc3e45.png)

**Comment by mk-pmb** [comment]
Good idea! I only need the corner pillars, which I'd find more related to `//walls` (because of the missing top and botton), but I can live with that option on `//edges` as well.

**Comment by TomyLobo** [comment]
well, we can't offer a command for every corner case (pun intended).
flags could work, but what about just the corners? what about just the bottom corners? what about just corner pillars but rotated 90°?

There's always a case missing and in order to still support flexible shape generation, we have //generate.
If you can't do the math for that and you don't have anyone you can ask about it, you can just select the 4 corner pillars separately and draw them one by one.

Take a look at that earlier command, btw:
```
//g magenta_concrete (abs(x)==1)+(abs(y)==1)+(abs(z)==1)>=2
```
What happens here is that we count the number of outer walls this block is a part of.
If a block is part of 3 walls, it's a corner.
If a block is part of 2 walls, it's an edge.
If a block is part of 1 wall, it's a face.
Grab a (real-life) cube, pick a corner/edge/face and count the number of walls it's bordering.
You'll arrive at those same numbers.

Now in order to only draw the corner pillars, we need to adjust the formula slightly.
Basically, we want to completely ignore the top and bottom (i.e. positive and negative y) walls in this calculation, so we just remove the `(abs(y)==1)` term from the expression:
```
//g magenta_concrete (abs(x)==1)+(abs(z)==1)>=2
```

Hope this helps in understanding what's going on.

**Comment by mk-pmb** [comment]
>  what about just the corners? what about just the bottom corners? what about just corner pillars but rotated 90°?

Good ideas. I suggest a combination of uppercase flags NSWETB that, if any of them are present, limiting the effect to the {north,south,west,east,top,bottom}-most blocks.
